test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,58 @@
+const agent = require('superagent');
+const statusCode = require('http-status-codes');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('First Api Tests', () => {
+
```

Comment:
Dentro de la función de `describe` deben ir los `it` que tienes abajo. La razón de esto, es que un **describe** se utiliza como una definición de una suite (conjunto) de pruebas que tienen algo en común (probar la url https://httpbin.org/) y cada **it** corresponde a una prueba especifica.